lib/helpers.js

```diff
@@ -560,3 +565,46 @@ exports.getBluemixDefaultPlan = function (serviceType) {
     default: return 'Lite'
   }
 }
+
+function loadHttpAsync (uri) {
+  debug('in loadHttpAsync')
+
+  return requestAsync({ method: 'GET', uri: uri })
+    .catch(err => {
+      debug('get request returned err:', err)
+      throw new Error(chalk.red('failed to load swagger from:', uri, 'err:', err.message))
```

Comment:
Should probably change the error message here, or even just allow the error to bubble up without catching it at this level.